app/components/Footer.js

```diff
@@ -107,15 +107,19 @@ export default {
               <li>
                 <a class="footer-icon" href="https://www.facebook.com/PULibrary">
                   <img :src="facebook">
+                  <span class="visually-hidden">Facebook</span>
```

Comment:
I think adding "Facebook" as an alt property for the image would also resolve the issue -- would you be able to give that a try?

Reply:
Done!